### PR TITLE
[query] Don't call into JVM for parallel/export_type

### PR DESCRIPTION
**Pull request by cseed**
Use strings instead of ints.

### CODE REVIEW

File: hail/python/hail/linalg/blockmatrix.py [hunk 1]
```diff
@@ -1819,7 +1819,9 @@ def export(path_in, path_out, delimiter='\t', header=None, add_index=False, para
         """
         jrm = Env.hail().linalg.RowMatrix.readBlockMatrix(Env.backend()._jhc, path_in, joption(partition_size))
 
-        export_type = Env.hail().utils.ExportType.getExportType(parallel)
+        if parallel is None:
+            parallel = 'concatenated
```
Comment (tpoterba) [inline hail/python/hail/linalg/blockmatrix.py:1823]: syntax error

Comment (cseed) [reply hail/python/hail/linalg/blockmatrix.py]: Fixed.

File: hail/python/test/hail/test_ir.py [hunk 2]
```diff
@@ -101,13 +101,13 @@ def value_irs(self):
...
             ir.TableWrite(table, ir.TableNativeWriter(new_temp_file(), False, True, "fake_codec_spec$$")),
-            ir.TableWrite(table, ir.TableTextWriter(new_temp_file(), None, True, 0, ",")),
+            ir.TableWrite(table, ir.TableTextWriter(new_temp_file(), None, True, "concatenated", ",")),
             ir.MatrixAggregate(matrix_read, ir.MakeStruct([('foo', ir.ApplyAggOp('Collect', [], [ir.I32(0)]))])),
             ir.MatrixWrite(matrix_read, ir.MatrixNativeWriter(new_temp_file(), False, False, "", None, None)),
             ir.MatrixWrite(matrix_read, ir.MatrixNativeWriter(new_temp_file(), False, False, "",
                                                               '[{"start":{"row_idx":0},"end":{"row_idx": 10},"includeStart":true,"includeEnd":false}]',
                                                               hl.dtype('array<interval<struct{row_idx:int32}>>'))),
-            ir.MatrixWrite(matrix_read, ir.MatrixVCFWriter(new_temp_file(), None, False, None)),
+            ir.MatrixWrite(matrix_read, ir.MatrixVCFWriter(new_temp_file(), None, 'concatenate', None)),
```
Comment (johnc1231) [inline hail/python/test/hail/test_ir.py:110]: Seems like you're maybe missing a "d" here? Like, should be "concatenated"? Unless I'm misreading aand this function takes a different set of strings than the above. This kind of makes me wish we had a Python equivalent of `ExportType.scala` so we could write like `ExportType.CONCATENATED` or something here and avoid spelling error bugs. Or maybe we have a way of specifying a more specific type than `str`? Like `oneof('concatened', 'parallel', ....)`?

Comment (cseed) [reply hail/python/test/hail/test_ir.py:110]: Excellent suggestions.  Done.  Much cleaner.

**Pull request merged**